privaterelay/templates/premium-promo.html

```diff
@@ -0,0 +1,254 @@
+                            <img class="c-brand-title" src="{% static 'images/logos/logo-firefox-relay.svg' %}">
+                            <p class="c-price non-premium-price">{% ftlmsg 'premium-promo-pricing-free-price' %}</p>
+                            <div class="c-features non-premium-list">
+                                <ul class="c-list">
+                                    <li>{% ftlmsg 'landing-pricing-free-feature-1' %}</li>
+                                    <li>{% ftlmsg 'landing-pricing-free-feature-2' %}</li>
+                                </ul>
+                            </div>
+                            <a
+                                class="mzp-c-button mzp-t-product mzp-t-secondary mzp-t-lg"
+                                href="{% provider_login_url 'fxa' process='login' %}"
+                                data-ga="send-ga-pings"
+                                data-event-category="Sign In"
+                                data-event-label="landing-pricing-free-cta"
+                            >
+                                {% ftlmsg 'landing-pricing-free-cta' %}
+                            </a>
+                        </div>
+                        <!--Premium Option-->
+                        <div class="c-option c-premium">
+                            <span class="c-premium-detail">{% ftlmsg 'landing-pricing-premium-price-highlight' %}</span>
+                            <img class="c-brand-title" src="{% static 'images/logos/logo-firefox-premium-relay.svg' %}">
+                            <p class="c-price premium-price">
+                                <span class="c-price-number">{{ monthly_price }}</span>
+                                <span class="c-currency">{% ftlmsg 'landing-pricing-premium-price' monthly_price="" %}</span>
+                            </p>
+                            <div class="c-features premium-list">
+                                <ul class="c-list">
+                                    <li>{% ftlmsg 'landing-pricing-premium-feature-1' %}</li>
+                                    <li>{% ftlmsg 'landing-pricing-premium-feature-2' %}</li>
+                                    <li>{% ftlmsg 'landing-pricing-premium-feature-3' %}</li>
+                                    <li>{% ftlmsg 'landing-pricing-premium-feature-4' %}</li>
+                                </ul>
+                            </div>
+                            <a
+                                class="mzp-c-button mzp-t-product mzp-t-secondary mzp-t-lg js-purchase-premium"
+                                href="{% premium_subscribe_url accept_language country_code %}"
+                                target="_blank" rel="noopener noreferrer"
+                                data-ga="send-ga-pings"
+                                data-event-category="Purchase Button"
+                                data-event-label="newlanding-plans-button"
```

Comment:
Should this be more page specific? Is it okay that it shares the same tags with the homepage?

Reply:
Well, I just happened to learn how we do analytics (i.e. how the `data-ga` attributes work) yesterday, but _after_ I submitted this PR. :) Fixed in 69c24ad7497f6795b2440a337f1dff2f25f0f7f8.